### PR TITLE
(SEC-728) update dependencies

### PR DESCRIPTION
**Pull request by lynsaywright**
**Description**
Update Dependencies to address Snyk vulnerabilities
update the following to the latest:
golang.org/x/net
gopkg.in/yaml.v2

https://tickets.puppetlabs.com/browse/SEC-728

### CODE REVIEW

File: pkg/orch/command_test.go [hunk 1]
```diff
@@ -104,7 +104,7 @@ func TestCommandScheduleTask(t *testing.T) {
 
 func TestCommandScheduleTaskWithScheduleOptions(t *testing.T) {
 
-	var options *ScheduleOptions = NewScheduleTaskOptions(time.Duration(24) * time.Hour)
+	var options = NewScheduleTaskOptions(time.Duration(24) * time.Hour)
```
Comment (eoinmcq) [inline pkg/orch/command_test.go:107]: shouldn't need the var before it

File: pkg/orch/command_test.go [hunk 2]
```diff
@@ -164,8 +164,8 @@ var expectedCommandScheduleTaskResponse = &ScheduledJobID{ScheduledJob: struct {
 
 func TestNewScheduleTaskOptions(t *testing.T) {
 
-	var optionsHour *ScheduleOptions = NewScheduleTaskOptions(time.Duration(1) * time.Hour)
-	var optionsMinutes *ScheduleOptions = NewScheduleTaskOptions(time.Duration(60) * time.Minute)
+	var optionsHour = NewScheduleTaskOptions(time.Duration(1) * time.Hour)
```
Comment (eoinmcq) [inline pkg/orch/command_test.go:167]: out of interest why have you changed these lines?

Comment (lynsaywright) [reply pkg/orch/command_test.go:167]: linting warnings - but can rollback?

Comment (eoinmcq) [reply pkg/orch/command_test.go:167]: as long as the build still works. I thought the linter issues would have broken it but could be wrong.

**Review by eoinmcq — APPROVED** [review]

**Pull request merged**